### PR TITLE
Adding myself to the site

### PR DESCRIPTION
**Pull request by sjn88**
Ready to merge. Checked it works locally but would love some feedback on the description.

### CODE REVIEW

File: _data/team.yml [hunk 1]
```diff
@@ -212,3 +212,14 @@
...
   github: chrisjeffery
+
+- id: sean
+  name: Sean Newham
+  image: /assets/img/custom/team/sean_newham.jpg
+  role: Software Craftsman
+  shortDescription: Sean became involved in Craftsmanship early in his career; writing a kata to introduce his local community to the joys of bridge (which he plays) and mocking. He is motivated by...
+  fullDescription: >
+    <p>Sean became involved in Craftsmanship early in his career; writing a kata to introduce his local community to the joys of bridge (which he plays) and mocking. He is motivated by a variety of problems. The current three are organisational structures and how they hinder us from making awesome, usable software; fixing teams, projects and software that are going badly and making an api that encourages good development practices from consumers.</p> 
+    <p>He believes in the devops philosophy and never leaves home without at least one way to write F# (his current laptop has 3 - mono, .net core and visual studio). His current interest in machine learning, due to an unhealthy obsession with matrices that started in his degree in Natural Sciences, specialising in Material Science and Metallurgy.</p>
```
Comment (dhatanian) [inline _data/team.yml:223]: Missing verb in `His current interest in machine...`

File: _data/team.yml [hunk 2]
```diff
@@ -212,3 +212,14 @@
...
   twitter: Chris_Jeffery
   github: chrisjeffery
+
+- id: sean
+  name: Sean Newham
+  image: /assets/img/custom/team/sean_newham.jpg
+  role: Software Craftsman
+  shortDescription: Sean became involved in Craftsmanship early in his career; writing a kata to introduce his local community to the joys of bridge (which he plays) and mocking. He is motivated by...
```
Comment (alvarogarcia7) [inline _data/team.yml:220]: Why add the last, incomplete sentence?

File: _data/team.yml [hunk 3]
```diff
@@ -212,3 +212,14 @@
...
+  fullDescription: >
+    <p>Sean became involved in Craftsmanship early in his career; writing a kata to introduce his local community to the joys of bridge (which he plays) and mocking. He is motivated by a variety of problems. The current three are organisational structures and how they hinder us from making awesome, usable software; fixing teams, projects and software that are going badly and making an api that encourages good development practices from consumers.</p>
```
Comment (alvarogarcia7) [inline _data/team.yml:222]: Craftsmanship or software craftsmanship?

Comment (jukecraft) [inline _data/team.yml:222]: The last sentence is a bit long. Consider splitting it.

File: _data/team.yml (hunk 1 above)
Comment (jukecraft) [reply _data/team.yml:223]: I'd separate the devops and the f# part.
Without a way to....  Sounds a bit weird - maybe say "without a development environment"?

File: _data/team.yml (hunk 3 above)
Comment (jukecraft) [reply _data/team.yml:222]: Which community?

Comment (jukecraft) [reply _data/team.yml:222]: The current _top_ three?

**Comment by stevelydford** [comment]
Not **_all_** organisational structure, or at least every part of any given organisational structure, is a hindrance. I would consider changing that phrase to something more like: 

organisational structures and how they affect the way we build.....

or

the affect of organisational structures on the successful delivery of.....

something like that anyway ;)

**Comment by sjn88** [comment]
I'll take your comments and send forth another version.

**Pull request merged**